.coafile

```diff
@@ -12,12 +12,22 @@ use_spaces = True
 # will be executed sequentially; also we need the LineLengthBear to double
 # check the line length because PEP8Bear sometimes isn't able to correct the
 # linelength.
-bears = SpaceConsistencyBear, PyUnusedCodeBear, QuotesBear
+bears = SpaceConsistencyBear, QuotesBear
 language = Python
 preferred_quotation = '
 
 default_actions = *: ApplyPatchAction
 
+[python_imports]
+# PyUnusedCodeBear is run separately beacause we need to check for unused 
+# imports. Do not set default_action to ApplyPatchAction as it may lead to some 
+# required imports being removed that might result in coala behaving weirdly.
+
+bears = PyUnusedCodeBear
+language = Python
+
+remove_all_unused_imports = true
```

Comment:
Hey sorry, this fell out of my notifications, but what I mean is that `PyUnsedCodeBear` has another setting `remove_all_unused_variables` and we should account for that. Other explicitly switch it off, or just explain that it also removes unused variables.